jekyll/_cci2/private-images.md

```diff
@@ -20,7 +30,6 @@ jobs:
     working_directory: ~/my_app
     steps:
       - checkout
-      - setup_remote_docker
```

Comment:
Removing setup_remote_docker means we can't use the subsequent `docker` commands.